helsingborg-stad/app-mitt-helsingborg
# Fix/pdf upload

## Explain the changes you’ve made
Make it possible to upload PDF together with other files like images.

Also did smaller refactoring.

## Explain why these changes are made
A user needs to be able to upload PDF when answering the form, hence adding the ability to do so.

## Explain your solution
Added new properties to the pdfUpload component to make it more alike the imageUploader component. This to make it easier in the future to refactor the file selector components to only one. 
Also by doing so we can remove the uploading logic from both pdfUploader and imageUploader components to fileUploader components instead.

## How to test

1. Checkout this branch
2. Fire upp the simulator by running the command `yarn ios` or `yarn android`
3. Make sure you have a form that supports both imageUploader and PdfUploader
4. Add files and images (or only one of them)
5. When uploading the files, check the number of files uploaded in the uploading info modal

## Tested environments

- [] Storybook on a iOS device/simulator.
- [] Building the Application on a Android device/simulator.

## Screenshots
![simulator_screenshot_C76E233A-4538-4083-9AE0-3F3246DF764E](https://user-images.githubusercontent.com/81250970/159956068-3690adf4-bab4-4353-bb76-6c6113386a65.png)
